cluster/addons/metadata-agent/stackdriver/metadata-agent.yaml

```diff
@@ -7,6 +7,19 @@ metadata:
     kubernetes.io/cluster-service: "true"
     addonmanager.kubernetes.io/mode: Reconcile
 ---
+apiVersion: v1
+kind: ConfigMap
+metadata:
```

Comment:
This ConfigMap is missing labels for addon manager:
kubernetes.io/cluster-service: "true"
addonmanager.kubernetes.io/mode: Reconcile